docs/src/HowToGuides/BalanceLaws/how_to_define_bcs.md

```diff
@@ -0,0 +1,52 @@
+# How to define boundary conditions
+
+```@meta
+CurrentModule = ClimateMachine.BalanceLaws
+```
+
+## Discontinuous Galerkin (DG) method
+
+Boundary conditions (BCs) are defined _weakly_ in ClimateMachine's
+DG implementation. We do this by defining BCs _through_ the _fluxes_,
+rather than on the prognostic variables themselves. Therefore, BCs
+are enforced by prescribing fluxes (numerical fluxes, in DG terminology)
+that drive the solution at the boundary to satisfy our desired BCs.
+
+One major advantage of this implementation approach is that BCs are
+handled in phase space, so we only need one implementation for all
+time steppers.
+
+In the context of the PDE that `ClimateMachine.jl` solves:
+
+`R(Y, t) = ∂_t Yᵢ + (∇•(F₁(Y) + F₂(Y,G)))ᵢ - (S(Y,G))ᵢ = 0, G = ∇Y`,
+
+the following functions must be defined for a
+[`BalanceLaw`](@ref) to fully define BCs:
+
+```@meta
+CurrentModule = ClimateMachine.DGMethods.NumericalFluxes
+```
+
+| **Method**                                       | Internal/User-facing  |  If present | Calls to user-facing functions |
+|:-----|:-----|:-----|:-----|
+| [`numerical_boundary_flux_first_order!`](@ref)   | User-facing  | F₁                  | `boundary_state!(::NumericalFluxFirstOrder)`, `numerical_flux_first_order!(::NumericalFluxFirstOrder)`    |
+| [`numerical_boundary_flux_second_order!`](@ref)  | User-facing  | F₂                  | `normal_boundary_flux_second_order!`[, `boundary_flux_second_order!` ,`boundary_state!(::NumericalFluxSecondOrder)`, `flux_second_order!`]  |
```

Comment:
I believe in the code the (normal) `local_flux` = `fluxTn` = F1+F2 = the total flux, so when you're modifying it in the `numerical_boundary_flux_second_order` you need to be careful not to overwrite the first flux contribution (unless that's what you want to do, i.e. impose the Robin BC on the total flux)

Reply:
Maybe this would be a good footnote somewhere?